src/Stratis.StratisD/Program.cs

```diff
@@ -28,7 +28,7 @@ public static void Main(string[] args)
             try
             {
                 Network network = args.Contains("-testnet") ? Network.StratisTest : Network.StratisMain;
-                NodeSettings nodeSettings = new NodeSettings("stratis", network, ProtocolVersion.ALT_PROTOCOL_VERSION).LoadArguments(args);
+                NodeSettings nodeSettings = new NodeSettings("stratis", network, ProtocolVersion.ALT_PROTOCOL_VERSION).LoadArguments(args, true);
```

Comment:
So here you would go:

`var nodeSettings = new NodeSettings("stratis", network, ProtocolVersion.ALT_PROTOCOL_VERSION).LoadArguments(args).LoadConfiguration();`

Reply:
Perhaps not in this case but yes.